supportedBackends/aws/src/main/scala/cromwell/backend/impl/aws/AwsBatchAsyncBackendJobExecutionActor.scala

```diff
@@ -362,21 +405,25 @@ class AwsBatchAsyncBackendJobExecutionActor(override val standardParams: Standar
 
   override def globParentDirectory(womGlobFile: WomGlobFile): Path =
     configuration.fileSystem match {
-      case  AWSBatchStorageSystems.s3 => {
+      case  AWSBatchStorageSystems.s3 =>
         val (_, disk) = relativePathAndVolume(womGlobFile.value, runtimeAttributes.disks)
         disk.mountPoint
-      }
       case _ => commandDirectory
     }
 
-      override def isTerminal(runStatus: RunStatus): Boolean = {
+  override def isTerminal(runStatus: RunStatus): Boolean = {
     runStatus match {
       case _: TerminalRunStatus => true
       case _ => false
     }
   }
 
+  /**
+    * Asynchronously upload the command script to the script path
+    * @return a `Future` for the asynch operation
+    */
   def uploadScriptFile(): Future[Unit] = {
+    //todo possibly remove this method as we don't use the script
```

Comment:
I'm going to open a ticket to deal with this in the future. We do use a script but we do it in a different way so this part needs removal but removing it will require some other changes.